### DISCUSSION TITLE
crontab speedtest gives lower values then running from the admin page.

### DISCUSSION THREAD

**Discussion by henkiewie** [alexjustesen/speedtest-tracker#1611] [Q&A] [answered]
Well the title says it all.
I do a speedtest on the same server every day. (through env variables)
When I do a speedtest by hand (same server) right afer or before the crontab time, I always get higher values.
Is this a known bug?

I have another docker container also running a daily ookla speedtest (dashdot) which are also giving the higher values. 
Maybe something to with the crontab priority in the docker container or is the crontab job using a different (slower) URL?

**Comment by alexjustesen** [comment, marked as answer]
This has come up in the past in some issues and isn't a bug of the application but an indication of network or speedtest server congestion. Avoid the top or bottom of the hour and if that doesn't help choose a different server.

Fundamentally there is no process difference between a scheduled test and a manual test.

**Reply by henkiewie** [reply]
OK thanks for the answer. Good to know.

**Reply by henkiewie** [reply]
Just to illustrate your point. Every on the hour check the number is going down.

<img width="1294" alt="Screenshot 2024-07-18 at 14 19 25" src="https://github.com/user-attachments/assets/5e78b755-c17a-4642-902d-c15c13aa02a3">

**Reply by alexjustesen** [reply]
Thanks for sharing a data point for others to reference.
